scripts/release/post-release.sh

```diff
@@ -2,6 +2,37 @@
 
 set -e
 
-# publish binaries to bintray
-scripts/publish/deb.sh
-scripts/publish/rpm.sh
+cloudsmith whoami
```

Comment:
Is this intentionally?

Reply:
It was but not super obvious why. Will add a comment